gensim/models/ldamodel.py

```diff
@@ -44,7 +44,27 @@
 from six.moves import xrange
 import six
 
-# log(sum(exp(x))) that tries to avoid overflow
+FAST_VERSION = 0
```

Comment:
Follow same approach as in word2vec: from gensim.models.word2vec_inner import FAST_VERSION